code/modules/clothing/spacesuits/_spacesuits.dm

```diff
@@ -1,4 +1,5 @@
-#define THERMAL_REGULATOR_COST 18 // the cost per tick for the thermal regulator
+/// Charge per tick consumed by the thermal regulator
+#define THERMAL_REGULATOR_COST (18 KILO WATTS)
```

Comment:
If it's per tick, then it's joules since it's not scaled by time. If it gets scaled by time, then the description of the define should get changed.

Reply:
the define for `JOULES` and `WATTS` should say when to use which because it's not very clear to someone who hasn't been in a physics classroom in 8 years

Reply:
I already did though.